SPECS/binutils/binutils.spec

```diff
@@ -16,8 +37,29 @@ Patch2:         thin_archive_descriptor.patch
 Patch3:         CVE-2021-45078.patch
 Patch4:         CVE-2022-38533.patch
 Patch5:         CVE-2022-4285.patch
+# The gold linker doesn't understand the 'module_info.ld' script passed to all linkers and the tests fail to correctly link.
+Patch6:         disable_gold_test.patch
 Provides:       bundled(libiberty)
 
+Requires:       %{name}-common = %{version}-%{release}
+
+# Moving macro before the "SourceX" tags breaks PR checks parsing the specs.
+%global do_package() \
+%if %2 \
```

Comment:
How are the %1 and %2 variables set? Is there a new build option to enabel cross building?

Reply:
No new option, they have different values for AMD64 and ARM64 builds. Check the top of the file - that's where the arguments passed to `do_package` are defined for each architecture.

Reply:
These are macro arguments and the macro is used slightly lower to conditionally define a subpackage in this spec. Right now it's only to build `binutils-aarch64-linux-gnu` for the AMD64 architecture but I decided to follow Fedora's idea with these macros to make it easier to add new subpackages in the future and control, which architecture builds which cross-compilation components.